ic-task-scheduler/src/scheduler.rs

```diff
@@ -214,6 +221,15 @@ where
         });
     }
 
+    /// Returns the next task id.
+    fn next_task_id(&self) -> u64 {
+        let mut lock = self.task_id_sequence.lock();
+        let id = *lock.get();
+        lock.set(id + 1)
+            .expect("Unable to access the stable storage to set the next poll id");
```

Comment:
task in place of poll.

Anyway, should we worry about overflowing u64 with an intense use over the time of the scheduler?

Reply:
By processing 1 billion tasks a day, it would take 50 million years to overflow  😶